### DISCUSSION TITLE
Enable/Disable Channel-Access for Users on HmIP-WKP and HmIP-DLD

### DISCUSSION THREAD

**Discussion by SeriousPat** [SukramJ/aiohomematic#1082] [Ideas]
(EN) There are the Switches Missing in HA for the Channel-Access on HmIP-WKP and HmIP-DLD.
![image](https://github.com/danielperna84/hahomematic/assets/98743842/83135dd9-09c2-463f-8b78-78c267c1e1fc)

(DE) Beim Keypad (HmIP-WKP) und der Keymatic (HmIP-DLD) kann man 8 User einrichten und diese Berechtigen. Bedeute auf diese Art kann man die Zugangspins aktivieren oder deaktivieren. In HA sieht man den Zustand 
![image](https://github.com/danielperna84/hahomematic/assets/98743842/6b28d5e6-5d82-4857-82cf-f78c4f839f5c)
 , aber es fehlt die Möglichkeit diese auch zu schalten. Ich sehe etwas mit "press lock/unloch ch1-16" aber es wird nicht erkannt.
![image](https://github.com/danielperna84/hahomematic/assets/98743842/05b4da88-9900-465c-9356-e9dddb8fe5c1)
![image](https://github.com/danielperna84/hahomematic/assets/98743842/7412a996-a332-447b-8586-85cb84e59281)

**Comment by gravidigit** [comment]
(DE) Habe das selbe Thema mit dem HmIP-WKP, gibt es hier schon neue Erkentnisse? Danke!
